h/services/nipsa.py

```diff
@@ -49,8 +72,14 @@ def unflag(self, user):
         queue).
         """
         user.nipsa = False
+        if self._flagged_userids is not None:
```

Comment:
Would you ever call unflag on a user that is shadowbanded where the _flagged_userids hasn't been populated yet? If so I think we have a bug here.